src/src/hw7/hillel/Animals.java

```diff
@@ -0,0 +1,41 @@
+package hw7.hillel;
+
+public class Animals {
```

Comment:
The class should be renamed to `Animal`. Please use it as a rule, that name of a class shouldn't be plural but singular (`Animals` is wrong, `Animal` is correct)